extensions/jfr/runtime/src/main/java/io/quarkus/jfr/runtime/SpanIdRelational.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.jfr.runtime;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import jdk.jfr.Description;
+import jdk.jfr.Label;
+import jdk.jfr.MetadataDefinition;
+import jdk.jfr.Name;
+import jdk.jfr.Relational;
+
+/**
+ * This is an annotation that associates multiple events in JFR by SpanId.
+ * Fields that can be annotated with this annotation must be in the String class.
+ */
+@Relational
+@MetadataDefinition
+@Name("io.quarkus.SpanId")
+@Label("Span ID")
+@Description("Link events that have the common Span ID")
```

Comment:
In the OpenJDK project, we have tried to avoid using the word event in labels and descriptions. You might want to do the same.

The rationale is that data may be pushed into a database or visualized in ways where the event concept is no longer visible to the user. I would drop the description, since it is pretty obvious, or keep it more abstract. i.e. "Links spans with the same ID together". Same for the TraceID.

Reply:
Thanks for the advice.
I have fixed both Trace and Span.

- https://github.com/quarkusio/quarkus/blob/49f38e3fcb4174fae042fdc1d136f3485a8dd43f/extensions/jfr/runtime/src/main/java/io/quarkus/jfr/runtime/TraceIdRelational.java
- https://github.com/quarkusio/quarkus/blob/49f38e3fcb4174fae042fdc1d136f3485a8dd43f/extensions/jfr/runtime/src/main/java/io/quarkus/jfr/runtime/SpanIdRelational.java